modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase

```diff
@@ -1017,6 +1017,35 @@ definition {
 			locator1 = "MyWorkflowTasks#MY_WORKFLOW_TASKS_SEARCH_RESULT_BAR");
 	}
 
+	@description = "LPS-172352 - Verify if it is possible to specify the recipient type in a workflow task"
+	@priority = 4
+	test CanSpecifyTheRecipientTypeInWorkflowTask {
+		task ("Given that a user accesses the workflow") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Workflow",
+				panel = "Applications",
+				portlet = "Process Builder");
```

Comment:
@thaiscabrals let's access Process Builder via UI to make our tests faster

Reply:
Did you mean via API? We're already accessing it via UI.

Reply:
sorry I meant URL! @thaiscabrals